ingestion_tools/scripts/data_validation/fixtures/data.py

```diff
@@ -218,17 +218,29 @@ def tomogram_metadata(tomo_meta_file: str, filesystem: FileSystemApi) -> Dict:
     with filesystem.open(tomo_meta_file, "r") as f:
         return json.load(f)
 
+@pytest.fixture(scope="session")
+def all_vs_tomogram_metadata(voxel_dir: str, filesystem: FileSystemApi) -> list[Dict]:
+    """Load all tomogram metadata for this voxel spacing."""
+    metadatas = []
+    for item in filesystem.glob(os.path.join(voxel_dir, "Tomograms/*/tomogram_metadata.json")):
+        with filesystem.open(item, "r") as f:
+            metadatas.append(json.load(f))
+    return metadatas
+
 
 # ==================================================================================================
 # Neuroglancer fixtures
 # ==================================================================================================
 
 
 @pytest.fixture(scope="session")
-def neuroglancer_config(neuroglancer_config_file: str, filesystem: FileSystemApi) -> Dict:
+def neuroglancer_configs(neuroglancer_config_files: List[str], filesystem: FileSystemApi) -> Dict:
```

Comment:
Support multiple ng configs per voxelspacing

Reply:
I think there are things we can validate within the configs themselves, but this is also outside of the scope of this PR I believe.